src/installer/pkg/projects/Microsoft.DotNet.ILCompiler/Microsoft.DotNet.ILCompiler.pkgproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <GeneratePackage>true</GeneratePackage>
-    <IncludeSymbolsInPackage>true</IncludeSymbolsInPackage>
```

Comment:
Does this remove things like Runtime.ServerGC.pdb (on Windows) or System.Private.CoreLib.pdb (on all platforms)?

We deliberately want them in the package.

Reply:
Yes, we want .pdbs for runtime binaries in the non-symbol package.

If you remove the .pdbs for native object files from the package, we will end up with a ton of warnings during `dotnet publish`:
```
  Generating native code
bootstrapper.obj : warning LNK4099: PDB '' was not found with 'bootstrapper.obj' or at ''; linking object as if no debug
info [C:\repro\repro.csproj]
Runtime.WorkstationGC.lib(threadstore.cpp.obj) : warning LNK4099: PDB '' was not found with 'Runtime.WorkstationGC.lib(th
readstore.cpp.obj)' or at ''; linking object as if no debug info [C:\repro\repro.csproj]
Runtime.WorkstationGC.lib(EHHelpers.cpp.obj) : warning LNK4099: PDB '' was not found with 'Runtime.WorkstationGC.lib(EHHe
lpers.cpp.obj)' or at ''; linking object as if no debug info [C:\repro\repro.csproj]
....
```
The .pdbs for managed runtime binaries are not needed at the moment, but they will be needed for #68714

Reply:
Thanks. I will try to figure out how to only filter out .dbg from non-symbols package (keeping managed pdbs intact). FWIW, in `runtime.linux-x64.microsoft.dotnet.ilcompiler/9.0.0-preview.5.24272.2` package, total size of .pdb files is  9.2 MB.

Reply:
Changed such that `Microsoft.DotNet.ILCompiler.<Version>.nupkg` contains pdb and `.symbols.nupkg` contains  both pdb and dbg/dSYM/debug etc.